polylogue/cli/completions_cli.py

```
@@ -10,6 +12,38 @@
 
+_SUPPORTED_SHELLS = ("bash", "zsh", "fish")
+
+
+def _normalize_shell_name(value: str) -> str:
+    candidate = value.strip()
+    if not candidate:
+        return ""
+    try:
+        candidate = Path(candidate).name
+    except Exception:
+        pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)